app/views/books/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1> Welcome to the bookstore <%= current_user.email %></h1>
+<h1>Bookstore</h1>
+<p>
+  <%= form_tag(root_path, method: :get) do %>
+    <%= text_field_tag :search %>
+    <%= submit_tag "Search" %>
+  <% end %>
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Author</th>
+      <th>Published Date</th>
+      <th>Price</th>
+      <th>Number Purchased</th>
+    </tr>
+  <% @books.each do |book| %>
+    <tr>
+      <td><%= book.title %></td>
+      <td><%= book.author %></td>
+      <td><%= book.published_date %></td>
+      <td><%= book.price_cents %></td>
+    </tr>
```

Comment:
I think you forgot to add the "Number purchased" column in the body of this table.